deployment/ansible/group_vars/all

```diff
@@ -4,19 +4,19 @@ developing_or_testing: "'development' in group_names or 'test' in group_names"
 not_developing: "'development' not in group_names"
 not_testing: "'test' not in group_names"
 
-java_version: "7u71-2.5.3-0ubuntu0.12.04.1"
+java_version: "7*"
```

Comment:
Globbing right before the `u` makes me worried a little bit. I think adjusting this to `7u71*` could help keep things up-to-date, but mitigate concerns around changes across update versions. The Solr community is big on staying on a particular update and I believe they are one of the biggest reporters of JVM bugs.